packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -7,16 +7,48 @@ import Color from "@khanacademy/wonder-blocks-color";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
 
-export default class TextField extends React.Component<{||}> {
+type State = {|
+    enabled: boolean,
+    disabled: boolean,
+    error: ?string,
+    focused: boolean,
+|};
+
+export default class TextField extends React.Component<{||}, State> {
+    state: State = {
+        enabled: true,
+        disabled: false,
+        error: null,
+        focused: false,
+    };
+
+    handleOnFocus: () => void = () => {
```

Comment:
You would want to add flow types for the function signature here. Not needed right now, but it'd be good to have it in mind when you add the `onFocus` and `onBlur` props later.